crates/bevy_reflect/src/lib.rs

```diff
@@ -1426,6 +1426,26 @@ mod tests {
         assert!(info.is::<MyValue>());
     }
 
+    #[test]
+    fn should_permit_higher_ranked_lifetimes() {
```

Comment:
Nit: This is totally fine so no need to change it haha, but just wanted to point out that we also have the [bevy_reflect_compile_fail_tests](https://github.com/bevyengine/bevy/tree/main/crates/bevy_reflect_compile_fail_tests) crate to test that things pass or fail build (with `.pass.rs` and `.fail.rs` files).

Reply:
Information overload led me to narrow my editor's view to `bevy_reflect`, so I missed that crate's existence. Sorry.

Reply:
No worries! Just spreading awareness rather than critiquing haha